samples/highcharts/xaxis/minortickspermajor/demo.js

```diff
@@ -0,0 +1,10 @@
+Highcharts.chart('container', {
+    yAxis: {
```

Comment:
Add a descriptive title to the demo. For example `<em>minorTicksPerMajor</em> is 2`.